primus_vk.cpp

```diff
@@ -1066,6 +1069,16 @@ void ImageWorker::copyImageData(uint32_t index, std::vector<VkSemaphore> sems){
       throw std::runtime_error("Layouts don't match at all");
     }
     TRACE_PROFILING_EVENT(index, "memcpy start");
+    VkMappedMemoryRange rendered_range {
+      VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE,
+      NULL,
+      render_copy_image->mem,
+      0,
+      VK_WHOLE_SIZE
+    };
+    VkResult invalid_result = device_dispatch[GetKey(swapchain.device)].InvalidateMappedMemoryRanges(swapchain.device, 1, &rendered_range);
+    VK_CHECK_RESULT(invalid_result);
```

Comment:
I'd prefer if you inline the "invalid_result" variable, as you don't need it in a second location and this is consistently the style all over primus_vk.cpp. Could you additionally use named arguments, again for conforming with the with the overall style? Otherwise it's not obvious what the arguments mean. And please change `NULL` to `VK_NULL_HANDLE`.